docker/owlbot/nodejs/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Version 0.1.1
+# Version 0.2.0
```

Comment:
Is anything consuming this version number?

Reply:
No, it's just meant as a way to kick off the Cloud Build job. Eventually I'm picturing this metadata could be used upstream when the PR is created for the new docker container.